### DISCUSSION TITLE
@import in css files not work

### DISCUSSION THREAD

**Discussion by wangcongyi** [web-infra-dev/rspack#8421] [Q&A]
![image](https://github.com/user-attachments/assets/a1b23ff6-6107-4aa7-b244-18dc8bfb5650)

写在 css 文件中的 @import 无效果，放在其他文件 另外引入就可以了。

从 craco 里迁移过来使用 rspack 的

rsbuild.config.js
![image](https://github.com/user-attachments/assets/bedd008b-2e69-4b7c-ba5e-8e30fe55534a)

**Comment by menglingfei** [comment]
我也遇到了同样的问题，请问解决了么

**Reply by wangcongyi** [reply]
https://github.com/web-infra-dev/rsbuild/discussions/3945#discussioncomment-11302154
